platform-includes/configuration/allow-urls/flutter.mdx

```diff
@@ -0,0 +1,14 @@
+```dart
+import 'package:flutter/widgets.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+Future<void> main() async {
```

Comment:
```suggestion
```

imo we can omit the main() function and the import for all future code snippets (except for snippets in the getting started page), they are just a lot of noise